cpp/src/gandiva/configuration.h

```diff
@@ -38,17 +38,17 @@ class GANDIVA_EXPORT Configuration {
  public:
   friend class ConfigurationBuilder;
 
-  const std::string& byte_code_file_path() const { return byte_code_file_path_; }
+  const std::string& precompiled_bitcode() const { return precompiled_bitcode_; }
 
-  std::size_t Hash() const;
+  // std::size_t Hash() const;
   bool operator==(const Configuration& other) const;
   bool operator!=(const Configuration& other) const;
 
  private:
-  explicit Configuration(const std::string& byte_code_file_path)
-      : byte_code_file_path_(byte_code_file_path) {}
+  explicit Configuration(const std::string& precompiled_bitcode)
+      : precompiled_bitcode_(precompiled_bitcode) {}
 
-  const std::string byte_code_file_path_;
+  const std::string& precompiled_bitcode_;
```

Comment:
This is not likely to be useful. Let's remove the precompiled_bitcode_ completely ?

Reply:
I've removed it, much better now :) 
I was hesitating to remove it, wasn't sure about the plans.